.github/workflows/deploy-gh-pages.yml

```diff
@@ -29,6 +29,8 @@ jobs:
       run: npm install
     - name: npm run build
       run: npm run build
+    - name: Check CNAME file existence
+      run: cat $PUBLISH_DIR/CNAME
```

Comment:
Where do we define $PUBLISH_DIR?

Reply:
We define $PUBLISH_DIR by env
```yaml
jobs:
  build:
    name: Build and deploy gh-pages
    env:
        PUBLISH_DIR : ./dist
```

Reply:
I'd prefer to move the check after the "git commit" so that we don't need to append any prefix :)